### PR TITLE
Cult Fixes

### PR DESCRIPTION
**Pull request by Geevies**
* Fixed Manifest Ghost not working for cultists.
* Fixed cultists being unable to summon soulstones and constructs.

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

File: code/game/gamemodes/cult/runes/manifest_ghost.dm [hunk 1]
```diff
@@ -1,54 +1,64 @@
...
 		"<span class='danger'>You feel pain, as bonds formed between your soul and this homunculus break.</span>", \
 		"<span class='warning'>You hear a faint rustling.</span>")
-		D.dust()
-	return
+		apparition.dust()
+		apparition = null
+
+/obj/effect/rune/manifest/Destroy()
+	apparition_check()
+	. = ..()
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/cult/runes/manifest_ghost.dm]: This also has to null out the apparition. Also, for destroy: `return ..()`

File: code/game/gamemodes/cult/runes/summon_soulstone.dm [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/obj/effect/rune/summon_soulstone
+	can_talisman = TRUE
+
+/obj/effect/rune/summon_soulstone/do_rune_action(mob/living/user, obj/O = src)
+	if(istype(O, /obj/effect/rune))
+		user.say("N[pick("'","`")]ath reth sh'yro eth d'raggathnor!")
+	else
+		user.whisper("N[pick("'","`")]ath reth sh'yro eth d'raggathnor!")
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/cult/runes/summon_soulstone.dm]: But why?

**Pull request merged**